src/Microsoft.ML.Data/Transforms/FeatureContributionCalculationTransformer.cs

```diff
@@ -266,9 +227,57 @@ private Delegate GetValueGetter<TSrc>(DataViewRow input, int colSrc)
     }
 
     /// <summary>
-    /// Estimator producing a FeatureContributionCalculatingTransformer which scores the model on an input dataset and
-    /// computes model-specific contribution scores for each feature.
+    /// Computes model-specific per-feature contributions to the score of each input vector.
+    /// See the list of currently supported models below.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of floats |
```

Comment:
@sfilipi @shmoradims were we going to use Single instead of float?

Reply:
yes


---
In reply to: [276422052](https://github.com/dotnet/machinelearning/pull/3384#discussion_r276422052) [](ancestors = 276422052)

Reply:
yep, that's my recollection too. 

---
In reply to: [276470283](https://github.com/dotnet/machinelearning/pull/3384#discussion_r276470283) [](ancestors = 276470283,276422052)